src/python/pants/backend/shell/util_rules/shell_command_test.py

```diff
@@ -553,7 +555,8 @@ def test_shell_command_boot_script(rule_runner: RuleRunner) -> None:
     assert "bash" in res.argv[0]
     assert res.argv[1:] == (
         "-c",
-        (
+        "cd src && /bin/bash -c "
+        + shlex.quote(
```

Comment:
I'd like to also see a test without the `cd` prefix.. (as there'll be a superfluous space in there I'd like to get rid of :) )

Reply:
that's a good idea anyway, imho :)